databases/week2/food_villa.sql

```diff
@@ -0,0 +1,152 @@
+
+DROP DATABASE IF EXISTS food_villa;
+
+CREATE DATABASE IF NOT EXISTS food_villa;
+
+USE food_villa;
+
+CREATE TABLE customer (
+                          customer_id INT(11) AUTO_INCREMENT PRIMARY KEY,
+                          first_name VARCHAR(30) NOT NULL,
+                          last_name VARCHAR(30) NOT NULL,
+                          middle_name VARCHAR(30) NOT NULL,
+                          email VARCHAR(50) NOT NULL,
+                          phone_number VARCHAR(15) NOT NULL,
```

Comment:
Line 14 , good to use varchar/string datatype for phone_number, but is good to know why as we often use int for number ?